docs/dev/news/README.md

```diff
@@ -2,7 +2,10 @@
 
 Each file in this folder describes one change in Dafny since the latest release.
 
-Files in this directory are named `<PR number>.<kind>` and each contains release notes for one merged PR.  `kind` is `break` (for breaking changes), `feat` (for new features and enhancements) or `fix` (for bug fixes).  No need to include a link to the PR in the file: it will be added automatically.
+Files in this directory are named `<PR number>.<kind>` (e.g. `1234.fix`) or `<description>.kind` (`assign-such-that-null.fix`) and each contains release notes for one merged PR.  `kind` is `break` (for breaking changes), `feat` (for new features and enhancements) or `fix` (for bug fixes).  No need to include a link to the PR in the file: it will be added automatically.
```

Comment:
```suggestion
Files in this directory are named `<PR or issue number>.<kind>` (e.g. `1234.fix`) or `<description>.kind` (`assign-such-that-null.fix`) and each contains release notes for one merged PR.  `kind` is `break` (for breaking changes), `feat` (for new features and enhancements) or `fix` (for bug fixes).  No need to include a link to the PR in the file: it will be added automatically.
```